dom.bs

```diff
@@ -4982,6 +4982,13 @@ invoked, must run these steps:
     <tr><td>"<code>wheelevent</code>"<td>{{WheelEvent}}<td>[[!UIEVENTS]]
   </table>
 
+ <li>
+  If the user agent does not support the <var>constructor</var> interface, or has it disabled
+  in its current configuration, let <var>constructor</var> be null.
+
+  <p class=note>Typically user agents have the {{TouchEvent}} interface disabled in some
+  configurations.
```

Comment:
I see. I think it would be better then to where we talk about the initial value of constructor we check for that being undefined and if it's undefined we throw a NotSupportedError and mention that this can be the case for TouchEvent in a note.